tokens/test/erc-1155/ERC1155Rarible.test.js

```diff
@@ -26,385 +26,670 @@ contract("ERC1155Rarible", accounts => {
     erc1271 = await ERC1271.new();
   });
 
-  it("mint and transfer by minter, token create by Factory", async () => {
```

Comment:
it's better not to fix formatting of existing tests in the same commit with new tests

Reply:
or it's hard to understand which tests are new